.github/workflows/update-tests-description.yaml

```diff
@@ -11,6 +11,8 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.PAT_TOKEN }}
```

Comment:
where is that Personal Access Token coming from? Cant we avoid that? rather use a different way of doing it as that its linked to the user that generated and then after a while we lost track of it...

Reply:
Yes but I found no other ways, and moreover we already have a token (the same) used for GCP self-hosted runner. For the runner it might be possible to use webhooks but this will need more time consuming tests. We can check to remove it later I think. Anyway if the tests description update doesn't work anymore this is not something that will block us.

Reply:
Cool for me, I already mentioned it. If it breaks I wont be here to fix it so I dont care.... joking of course :)

Maybe adding that info to the readme.md so when it breaks we know how to proceed?

Reply:
More useful in the workflow file for the info I think, no? Not sure that someone will check in `tests/README.md` file in case of issue in the GH action.

Tell me what do you think it's better, I will then add a note and push a commit for this before merging.

Reply:
either way, as long as its written somewhere its good for me :=)

Reply:
Added with this [commit](https://github.com/rancher/elemental/pull/598/commits/aacc78f3014ae801ec4c9ea50837086a8a6b9964).